nat-lab/bin/mqtt_listener.py

```diff
@@ -0,0 +1,18 @@
+import paho.mqtt.client as mqtt
+import sys
+
+
+def on_message(_client, _userdata, message):
+    print(f"{message.payload.decode()}")
+    sys.exit(0)
+
+
+mqttc = mqtt.Client(client_id="reciever", protocol=mqtt.MQTTv311)
+
+mqttc.on_message = on_message
+
+# TODO: add credentials when LLT-5604 will be ready
```

Comment:
IMHO for credentials, for MVP version, and keeping in mind that this is testing-only environment, we can hardcode something then make sure, the same is returned by API, and verify that those hardcoded values are provided by the clients in here.

Reply:
Here is a bit more info about API returning credentials: https://github.com/NordSecurity/libtelio/pull/815/files#r1766352606

Reply:
Oh we agreed on making this a separate PR/task. NVM then.